lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -49,6 +52,65 @@ def self.define_class_methods(caller_class, position_column, touch_on_update)
 
       private
 
+      # Update recordset to start at the given position
+      define_singleton_method :shuffle_to do |position|
+        max = nil
+        if ActiveRecord::VERSION::MAJOR < 4
```

Comment:
I'd like to drop Rails < 4.2 support, so this code fork could go. I'll look to updating the testing harness (so it actually works) plus to remove the Rails 3.2 and 4.0, 4.1 support today so you can test against that.